docker-compose.yml

```diff
@@ -2,11 +2,12 @@ version: "3.5"
 
 services:
   cardano-node:
-    image: inputoutput/cardano-node:1.35.3-configs
+    image: inputoutput/cardano-node:1.35.4
     environment:
       NETWORK:
+      CARDANO_NODE_SOCKET_PATH: ${CARDANO_NODE_SOCKET_PATH:-/ipc/node.socket}
```

Comment:
Hm, not quite sure what that will provide? `/ipc/node.socket` is the local path in the context of docker, not really available for the docker host.

Reply:
Ok, maybe I misunderstand something, but without it I can't query a tip... Or?

For example like this I will query a tip:

`docker exec -ti wallet-new-cardano-node-1 /usr/local/bin/cardano-cli query tip --mainnet`

Reply:
Ah, ok, makes sense. But what's the reasoning for:
```
CARDANO_NODE_SOCKET_PATH: ${CARDANO_NODE_SOCKET_PATH:-/ipc/node.socket}
```
I suppose this would be enough:
```
CARDANO_NODE_SOCKET_PATH: /ipc/node.socket
```
(Don't think we want to inject different `CARDANO_NODE_SOCKET_PATH`)

Reply:
${CARDANO_NODE_SOCKET_PATH:-/ipc/node.socket} -- that basically says if we do not set up CARDANO_NODE_SOCKET_PATH variable, use /ipc/node.socket, for it. So technically if you setup it manually it will use whatever you pass to it, otherwise if empty or unset use /ipc/node.socket

Reply:
That's right. My question is why would you like to set up anything other than `/ipc/node.socket` which is hardcoded in docker compose. If you pass anything different you won't be able to check the tip of the node running in this particular running container anyway. In fact, it may lead to misbehavior if you already have `CARDANO_NODE_SOCKET_PATH` set up on your docker host.